services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -362,7 +366,7 @@ async def _remove_single_orphaned_service(
             except (UserNotFoundError, ValueError):
                 user_role = None
 
-            project_uuid = currently_opened_projects_node_ids[service_uuid]
+            project_uuid = interactive_service.get("project_id", "")
```

Comment:
what happens if project_uuid is blank now?

Reply:
see above